apps/cookbook/src/app/showcase/button-showcase/button-showcase.component.html

```diff
@@ -1,4 +1,22 @@
 <div class="example">
+  <kirby-header-actions [visibleButtons]="2">
```

Comment:
TODO: Remove after review

Only added for visual review, header actions will be described in the header cookbook entry when that is introduced.